docs/import-assistant.md

```diff
@@ -0,0 +1,67 @@
+ # Import Assistant
+ 
+Run powerful AI-enabled commands to assist with your import script development.
+
+### Options
+
+- `--url`: The URL of the page to analyze
+- `--outputPath`: The directory to save the import scripts
```

Comment:
Maybe `### Mandatory Options` on line 5?  And `#### Additional Options` for each command below (if there are any)?